rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverter.groovy

```diff
@@ -66,6 +70,7 @@ class PackageNameConverter {
   // For example: nflx-djangobase-enhanced-0.1-h12.170cdbd.all
   public static OsPackageName parseRpmPackageName(String fullyQualifiedPackageName) {
     OsPackageName osPackageName = new OsPackageName()
+    String versionDelimiter = BakeRequest.PackageType.RPM.getVersionDelimiter()
```

Comment:
ah, using the getter for both now. I'm a bit surprised that the former worked since it's marked private. I guess it's a bug of sorts. https://issues.apache.org/jira/browse/GROOVY-3010